src/main/java/gregtech/loaders/recipe/RecyclingRecipes.java

```diff
@@ -1,111 +1,400 @@
 package gregtech.loaders.recipe;
 
+import com.google.common.collect.ImmutableList;
 import gregtech.api.recipes.RecipeBuilder;
 import gregtech.api.recipes.RecipeMaps;
 import gregtech.api.unification.OreDictUnifier;
 import gregtech.api.unification.material.Material;
 import gregtech.api.unification.material.Materials;
+import gregtech.api.unification.material.properties.BlastProperty;
 import gregtech.api.unification.material.properties.PropertyKey;
+import gregtech.api.unification.ore.OrePrefix;
 import gregtech.api.unification.stack.ItemMaterialInfo;
 import gregtech.api.unification.stack.MaterialStack;
+import gregtech.api.unification.stack.UnificationEntry;
+import gregtech.api.util.GTUtility;
 import net.minecraft.item.ItemStack;
+import net.minecraft.util.Tuple;
 
-import java.util.ArrayList;
-import java.util.List;
+import javax.annotation.Nonnull;
+import java.util.*;
 import java.util.Map.Entry;
-import java.util.function.Consumer;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 import static gregtech.api.GTValues.L;
 import static gregtech.api.GTValues.M;
-import static gregtech.api.unification.material.info.MaterialFlags.FLAMMABLE;
+import static gregtech.api.unification.material.info.MaterialFlags.*;
 
 public class RecyclingRecipes {
 
-    public static void init() {
-        initializeArcRecyclingRecipes();
-    }
+    // TODO - Fix recipe order with some things (noticed Hermetic Casings)
```

Comment:
This should be addressed, but there might be an edge case still. Loading order is important for some recipes now, as if a later recipe uses an input (say, Steam Oven using Bronze Casings), unification data could be wrong

Reply:
I have not noticed issues with the Hermetic Casings when I check